src/prefect/environments/storage/docker.py

```diff
@@ -99,6 +99,9 @@ class Docker(Storage):
             if `stored_as_script=True`.
         - stored_as_script (bool, optional): boolean for specifying if the flow has been stored
             as a `.py` file. Defaults to `False`
+        - extra_dockerfile_commands (list[str], optional): list of unstructured Docker build commands
```

Comment:
No need for "unstructured" here.

```suggestion
        - extra_dockerfile_commands (list[str], optional): list of Docker build commands
```